sdk/CONTRIBUTING.md

```diff
@@ -0,0 +1,86 @@
+# SDK Contribution Guide
+
+This document serves as a guide for creating a new Dagger SDK, but also to help maintain consistency between existing ones.
+
```

Comment:
```suggestion

## What is a Dagger SDK?

A Dagger SDK contains everything to develop CI/CD pipelines for a particular platform. This may include:

* A client library
* Examples
* Documentation
* Platform-specific tooling

Dagger SDKs are open-source, and developed jointly with the community. Some SDKs are mature enough to be considered stable, while others are experimental. Our goal is to eventually offer stable and feature-complete SDKs for all platforms.

## Do I need a Dagger SDK to program Dagger?

No. Dagger SDKs are an optional convenience for programming the Dagger Engine. Under the hood, all SDKs make queries to the same [GraphQL API](https://docs.dagger.io/api). It's perfectly OK to write GraphQL clients in code directly, either using your platform's [GraphQL client implementation](https://graphql.org/code/), or even a raw HTTP client. You will still have access to all features of the Dagger Engine; the only difference will be a less polished developer experience.

## Can I develop my own SDK?

Yes, with a little effort you can develop your own SDK. If you think others may benefit from it, please consider contributing it as an experimental SDK, so that others can test it and help improve it.

## How do I contribute to an existing SDK?

FIXME

## How do I contribute a new SDK?

FIXME

## What's the difference between an experimental and stable SDK?

FIXME
```